cmd/helmExecute.go

```diff
@@ -28,17 +28,25 @@ func helmExecute(config helmExecuteOptions, telemetryData *telemetry.CustomData)
 		TargetRepositoryPassword:  config.TargetRepositoryPassword,
 		HelmCommand:               config.HelmCommand,
 		CustomTLSCertificateLinks: config.CustomTLSCertificateLinks,
+		DeploymentName:            config.DeploymentName,
+		PackageVersion:            config.PackageVersion,
 	}
 
 	utils := kubernetes.NewDeployUtilsBundle(helmConfig.CustomTLSCertificateLinks)
 
-	helmChart := config.ChartPath + "Chart.yaml"
-	nameChart, packageVersion, err := kubernetes.GetChartInfo(helmChart, utils)
-	if err != nil {
-		log.Entry().WithError(err).Fatalf("failed to get version in Chart.yaml: %v", err)
+	if len(config.DeploymentName) == 0 || len(config.PackageVersion) == 0 {
+		helmChart := config.ChartPath + "Chart.yaml"
+		nameChart, packageVersion, err := kubernetes.GetChartInfo(helmChart, utils)
```

Comment:
we could use the `versioning` package with call `GetCoordinates()`

Reply:
done